index.css

```diff
@@ -33,3 +33,54 @@ body {
   top: 626px;
   margin: 0px;
 }
+
+.old-man {
+    padding-top: 140px;
+    padding-left: 100px;
+}
+.old-man > .img-men {
+    width: 609px;
+    height: 403px;
+}
+
+.old-man > .header-2 {
+    height: 116px;
```

Comment:
Could you remove all widths and height for all text classes? Move it the dov styles?